src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -208,6 +210,8 @@ private static CSharpSyntaxContext CreateContextWorker(Workspace workspace, Sema
             // If the second dot was typed, we just insert two dots.
             var isRightSideOfNumericType = leftToken.IsNumericTypeContext(semanticModel, cancellationToken);
 
+            var isInArgumentList = targetToken.Parent.IsKind(SyntaxKind.ArgumentList);
```

Comment:
1. isArgumentListToken.
2. this only seems to work for normal argumetn lits, not attributes or indexers.  intentional?

Reply:
fixed